gui/src/Notebook/Packer.purs

```diff
@@ -6,17 +6,19 @@ import Notebook.Types
 import Cells.Types as Cells
 import Console.Types as Console
 import Data.Array
+import Node.Path (sep)
 
-initialNotebook :: Notebook
-initialNotebook = Notebook
-  { title: ""
-  , subtitle: ""
-  , author: ""
-  , date: ""
-  , cells: [] :: Array Cells.Cell
-  , console: ">"
-  , filepath: "Main.hs"
-  }
+-- initialNotebook :: Notebook
+-- initialNotebook = Notebook
+--   { title: ""
+--   , subtitle: ""
+--   , author: ""
+--   , date: ""
+--   , cells: [] :: Array Cells.Cell
+--   , console: ">"
+--   , filepath: "Main.hs"
+--   , loaded : true
+--   }
```

Comment:
Commented code shouldn't be pushed, that's why we use Git 😁 

If we ever need to come back to get an old version of the code, we just `git checkout` or browse the commits 😉